src/aura/CMP_RecTypes_Dropdown/CMP_RecTypes_Dropdown.cmp

```diff
@@ -0,0 +1,24 @@
+<aura:component >
+    <aura:handler event="c:STG_EVT_Save" action="{!c.saveSetting}"/>
+    
+    <aura:attribute name="isView" type="Boolean" default="true"/>
+    <aura:attribute name="recTypeName" type="String" />
+    <aura:attribute name="recTypeId" type="String" />
+    <aura:attribute name="recTypes" type="Object[]" />
+
+    <aura:attribute name="setting" type="String" />
+    
+    <div class="slds-col slds-size--1-of-2">
```

Comment:
Any reason to include this div in the component and not in the parent component that is calling it?